config/scheduler.yml

```diff
@@ -0,0 +1,15 @@
+---
+sync_users_from_bambo_api:
+  cron: "0 6 * * *"
+  class: SyncUsersJob
+  queue: users
+  description: "This job will sync the users every 30 seconds after 120 seconds after start"
```

Comment:
Is the description correct? I mean, evey 30 seconds after 120 seconds? shouldn't it be every day at 6 am???